rollbar/lib/traverse.py

```diff
@@ -1,30 +1,25 @@
 import logging
 
-try:
-    # Python 3
-    from collections.abc import Mapping
-    from collections.abc import Sequence
-except ImportError:
-    # Python 2.7
-    from collections import Mapping
-    from collections import Sequence
 
 from rollbar.lib import binary_type, iteritems, string_types, circular_reference_label
+from rollbar.lib.type_info import (
```

Comment:
I think this could probably use a comment for the same reason as the `Transform` class import.